src/main/java/de/eddyson/tapestry/react/ReactModule.java

```diff
@@ -67,14 +76,16 @@ public static void setupJSModules(final MappedConfiguration<String, JavaScriptMo
     configuration.add("react", new JavaScriptModuleConfiguration(assetSource.resourceForPath(reactAssetPathToUse)));
     configuration.add("react-dom", new JavaScriptModuleConfiguration(
         assetSource.resourceForPath(productionMode ? reactDomAssetPathProduction : reactDomAssetPath)));
+    
+    // for isomorphic server side rendering:
+    configuration.add("react-dom-server",new JavaScriptModuleConfiguration(assetSource.resourceForPath(productionMode ? reactDomServerAssetPathProduction : reactDomServerAssetPath)));
```

Comment:
If possible, please don't add this to the `ModuleManager`. We don't want react-dom-server available on the client side.